README.md

```diff
@@ -1 +1,76 @@
+
+**Model description preview short** - the text people see on a card on the [main page](https://modelplace.ai/) (just after the model name)
+
+**Model mame full** - on a model page - (e.g. Faster region-based convolutional neural network with ResNet-50 FPN backbone https://modelplace.ai/models/2)
+
+**Model name short** - on the [main page](https://modelplace.ai/) (e.g. Faster R-CNN).
+
+**Dataset name** - it's may be public data e.g. MSCOCO or Open images, or your internal dataset.
+
+**Preview image** - we will run a model on the image and will use its output for visualization. This image will be both on the [main page](https://modelplace.ai/) and on the [model page](https://modelplace.ai/models/2)
+
+**Licence** - licence type e.g. Apache 2.0, MIT, Proprietary and so on
+**Number of classes** 
+**Author** - your name or company name
+**Metrics** - accuracy of your model
+
+### Optional
+**Dataset link** - link to dataset that you use for training and testing of your model
+**Source link** - link to your project page or profile page or company page and so on
+**System requirements** - what system setup( e.g. number of CPU GPU RAM) we should use to serve your model.
+
+**Logo** - we will add it to the preview.   
+_Note: This should be svg(png is possible) image with circle shape and size 128x128 pixels at least._
+
+**Avatar image** - you can add your own logo(by default we use our).   
+_Note: This should be svg(png is possible) image with circle shape and size 128x128 pixels at least._
+
+## Evironment
+- Install venv  
+```python3.7 -m pip install virtualenv```
+- Create an empty virtual environment for python3.7  
+```python3.7 -m  virtualenv  venv```
+- Activate it  
+```source venv/bin/activate```
+- Install pytest and wheel  
+```python3.7 -m pip install pytest wheel```
+- Install git and git-lfs  
+```sudo apt install git git-lfs```
+- Clone the repo 
+```git clone https://github.com/opencv-ai/modelplace.git```
+
+## Example
+You can run [an example](pytorch_fastercnn) of a package
+- Go to package folder pytorch_fastercnn
+```cd pytorch_fastercnn```
+- Install the package  
+```python3.7 setup.py install develop```
+- Run tests  
+```pytest pytorch_fastercnn```
+
+Example of model usage you can find [here](https://modelplace.ai/models/2).
+
+## Package
+We use an abstraction to unify model serving. You should create [a wheel package](https://packaging.python.org/tutorials/packaging-projects/#creating-the-package-files) with our interfaces. You can find example [here](pytorch_fastercnn).
+
+To create a package follow the steps:
+- Copy the template folder
+- Extend model.py as it's done in [an example](pytorch_fastercnn/pytorch_fastercnn/model.py)
+- Update [the setup.py](template/setup.py)
+- Copy your checkpoints to [the template/checkpoints](template/template/checkpoints) folder
```

Comment:
Minor: Copy your checkpoints to the [template/checkpoints](template/template/checkpoints) folder

Reply:
Next time, please, write a comment if it's not fixed